lib/tabular_form_builder.rb

```diff
@@ -54,6 +54,11 @@ def #{selector}(field, options = {}, *args)
     class_eval src, __FILE__, __LINE__
   end
 
+  def label(method, text = nil, options = {}, &block)
+    options[:class] = 'form--label'
```

Comment:
@floriank this will override any other class specified. I'd suggest using the pattern below, for select fields (or improving it on it somehow)